farmfs/keydb.py

```diff
@@ -9,7 +10,7 @@
 from func_prototypes import typed, returned
 
 @returned(str)
-@typed(str)
+@typed(bytes)
```

Comment:
if checksum returns bytes, we need to look at when checksums are converted into paths to make sure we convert back.